### PR TITLE
Heads of staff will now have cat organs removed at roundstart

### LINKED ISSUE #31869: People that have mutant bodyparts can regain them trough genetics when they select a job that removes them
Step 1. Select mutant ear/tail part in character setup
Step 2. Roll a head role
Step 3. Have no ears/tail
Step 4. Head over to genetics
Step 5. Get monkified
Step 6. Get humanified
Step 7. Have your mutant bodyparts

### PR DESCRIPTION
**Pull request by ghost**
Also adds a  fun proc called ```remove_catshit``` to humans!

fixes #31869

:cl: More Robust Than You
fix: Heads of staff will now have cat organs removed at roundstart
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human_helpers.dm [hunk 1]
```diff
@@ -156,3 +156,13 @@
 			return FALSE
 
 	return .
+
+/mob/living/carbon/human/proc/cure_cancer()
```
Comment (lzimann) [inline code/modules/mob/living/carbon/human/human_helpers.dm]: as much fun it is, name the proc something that actually makes sense

Comment (kevinz000) [inline code/modules/mob/living/carbon/human/human_helpers.dm]: this proc is bugged it won't delete you if you're in game mrty

**Comment by XDTM** [comment]
Make them a separate species already

**Comment by MoreRobustThanYou** [comment]
@XDTM might as well port tajarans

File: code/modules/mob/living/carbon/human/human_helpers.dm [hunk 2]
```diff
@@ -156,3 +156,14 @@
 			return FALSE
 
 	return .
+
+/mob/living/carbon/human/proc/remove_catshit()
+	var/obj/item/organ/ears/cat/T = getorganslot(ORGAN_SLOT_TAIL)
```
Comment (ShizCalev) [inline code/modules/mob/living/carbon/human/human_helpers.dm]: Reversed

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/human_helpers.dm (hunk 2 above)
Comment (MoreRobustThanYou) [reply code/modules/mob/living/carbon/human/human_helpers.dm]: fixed

File: code/modules/mob/living/carbon/human/human_helpers.dm [hunk 3]
```diff
@@ -156,3 +156,14 @@
 			return FALSE
 
 	return .
+
+/mob/living/carbon/human/proc/remove_catshit()
+	var/obj/item/organ/ears/cat/E = getorganslot(ORGAN_SLOT_EARS)
```
Comment (KorPhaeron) [inline code/modules/mob/living/carbon/human/human_helpers.dm]: What if they have non cat mutant ears?

**Comment by KorPhaeron** [comment]
This doesn't really fix the underlying problem in that taping cat ears to your head edits your dna

Ugly hybrid system

**Comment by kevinz000** [comment]
please go

**Comment by Bawhoppen** [comment]
It should not let them be selected as heads in the first place. Please do it properly.

File: code/modules/mob/living/carbon/human/human_helpers.dm [hunk 4]
```diff
@@ -156,3 +156,14 @@
 			return FALSE
 
 	return .
+
+/mob/living/carbon/human/proc/remove_catshit()
```
Comment (kevinz000) [inline code/modules/mob/living/carbon/human/human_helpers.dm]: or maybe name it remove_catparts or something normal?

Comment (optimumtact) [reply code/modules/mob/living/carbon/human/human_helpers.dm]: dont' backseat maintain

File: code/modules/mob/living/carbon/human/human_helpers.dm (hunk 3 above)
Comment (optimumtact) [reply code/modules/mob/living/carbon/human/human_helpers.dm]: won't happen, but if it does then the policy will still be that they have to be fully human to be a head anyway

File: code/modules/mob/living/carbon/human/human_helpers.dm (hunk 4 above)
Comment (kevinz000) [reply code/modules/mob/living/carbon/human/human_helpers.dm]: @optimumtact isn't it somewhat bad to have procs with obscene language though?

Comment (RemieRichards) [reply code/modules/mob/living/carbon/human/human_helpers.dm]: there's a var ```fuck_you_dir``` in the code.
generally the code can be as rude as it wants, so long as none of it slips into in-game text.

Comment (Arianya) [reply code/modules/mob/living/carbon/human/human_helpers.dm]: Also whats backseat maintaining, its not like he said "name it remove_catparts or this isn't getting merged", he made a suggestion.

Citrus plz

**Comment by Slignerd** [comment]
Poor implementation of what's already an feature.

**Comment by Kraseo** [comment]
dude that proc name is so fucking epic my balls just dropped XDDDDDD

**Comment by The-clown-known-as-Pepe** [comment]
probably will be merged in broken as fuck state like cat organs because

le funni cat meme xd xd

File: code/modules/jobs/job_types/job.dm [hunk 5]
```diff
@@ -76,9 +76,7 @@
 		announce(H)
 
 	if(CONFIG_GET(flag/enforce_human_authority) && (title in GLOB.command_positions))
-		H.dna.features["tail_human"] = "None"
-		H.dna.features["ears"] = "None"
-		H.regenerate_icons()
+		H.remove_catshit()
```
Comment (The-clown-known-as-Pepe) [inline code/modules/jobs/job_types/job.dm]: we hippie now xd

**Comment by Qbopper** [comment]
next thing you know someone will add a bold text message that says "YOU ARE VALID" when you select cat parts

File: code/modules/jobs/job_types/job.dm (hunk 5 above)
Comment (Arianya) [reply code/modules/jobs/job_types/job.dm]: dont' backseat maintain

**Comment by bgobandit** [comment]
@Qbopper the old purrbation message said that

**Comment by Qbopper** [comment]
ok good I thought it did say something along those lines but forgot so I wasn't sure if the joke would work

**Pull request merged**